.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/spark-gov-relay"]
 	path = lib/spark-gov-relay
 	url = https://github.com/marsfoundation/spark-gov-relay
+[submodule "lib/spark-alm-controller"]
+	path = lib/spark-alm-controller
+	url = https://github.com/marsfoundation/spark-alm-controller
```

Comment:
nit: is there a reason not to specify commits here? it would make crafters need to explicitly change it when needed, but would also avoid unwanted changes whenever some submodule gets updated (blocking possible code injection vector, although most submodules are owned)

Reply:
The commits are set in the lib directory. You can do `git submodule update --init --recursive` to update them.

Reply:
feel free to close 👍